README.md

```diff
@@ -1 +1 @@
-# Hello-world
\ No newline at end of file
+This Hello Microverse project is my first project as a Microverse student. I will be setting up HTML and CSS linters alongside 3 files of which one is to be ignored by git. 
```

Comment:
- [ ] For all your projects you will do in Microverse, you should make a descriptive and professional README file, feel free to use the [Template provided by Microverse](https://github.com/microverseinc/readme-template), also remember to use only the sections necessary according to this project, you can remove "Run tests", "Deployment".


- [Suggestion] Instead "feature-1" you can use "create_sidebar", "added_comments_section" which are more descriptives. (The feature branch name depends on what you did in the project)